src/main/java/de/tum/in/www1/artemis/service/ExerciseService.java

```diff
@@ -738,13 +738,19 @@ private void setAssessmentsAndSubmissionsForStatisticsDTO(CourseManagementOvervi
         }
     }
 
+    public void validateGeneralSettings(Exercise exercise) {
+        validateScoreSettings(exercise);
+        validateDateSettings(exercise);
+    }
+
     /**
      * Validates score settings
      * 1. The maxScore needs to be greater than 0
      * 2. If the specified amount of bonus points is valid depending on the IncludedInOverallScore value
      *
      * @param exercise exercise to validate
      */
+
     public void validateScoreSettings(Exercise exercise) {
```

Comment:
If I am not mistaken, this can now be private?
```suggestion
    private void validateScoreSettings(Exercise exercise) {
```